README.md

```diff
@@ -137,8 +142,67 @@ lg()
     fi
 }
 ```
+
 Then `source ~/.zshrc` and from now on when you call `lg` and exit you'll switch directories to whatever you were in inside lazyigt. To override this behaviour you can exit using `shift+Q` rather than just `q`.
 
+## Configuration
+
+All Lazygit configurations are stored on `<userConfigDir>/jesseduffield/lazygit/config.yml`, where
+`userConfigDir` is your OS-dependent user configurations folder, namely:
+
+- Windows: `%APPDATA%` (`C:\Users\<User>\AppData\Roaming`)
+- Linux/BSDs: `${XDG_CONFIG_HOME}` (`${HOME}/.config`)
+- MacOSX: `${HOME}/Library/Application Support`
+
+This file is auto-generated on first run.
+
+You can also get the default values with `lazygit -c`, which returns something like:
+
+```
+gui:
+  ## stuff relating to the UI
+  scrollHeight: 2
+  scrollPastBottom: true
+  mouseEvents: false # will default to true when the feature is complete
+  theme:
+    activeBorderColor:
+      - white
+      - bold
+    inactiveBorderColor:
+      - white
+    optionsTextColor:
+      - blue
+  commitLength:
+    show: true
+git:
+  merging:
+    manualCommit: false
+  skipHookPrefix: 'WIP'
+  autoFetch: true
+update:
+  method: prompt # can be: prompt | background | never
+  days: 14 # how often a update is checked for
+reporting: 'undetermined' # one of: 'on' | 'off' | 'undetermined'
+confirmOnQuit: false
+```
+
+### Custom pull request URLs
+
+Some git provider setups (e.g. on-premises GitLab) can have distinct URLs for git-related calls and
+the web interface/API itself. To work with those, Lazygit needs to know where it needs to create
+the pull request. You can do so on your `config.yml` file using the following syntax:
+
+```yaml
+services:
```

Comment:
what are your thoughts on nesting this under the `git` key in the config?

Reply:
makes sense to me - I honestly haven't read through all the options when I first implemented this

Reply:
I read through the config docs and I don't think the `services` should be below `git`, as the latter is related to the git tool itself, whereas the former is lazygit-specific.